lib/bypass/utils.ex

```diff
@@ -3,7 +3,7 @@ defmodule Bypass.Utils do
 
   Application.load(:bypass)
 
-  if Application.get_env(:bypass, :enable_debug_log, false) do
+  if Application.compile_env(:bypass, :enable_debug_log, false) do
```

Comment:
this will break the enable_debug_log macro. maybe this is better? https://github.com/PSPDFKit-labs/bypass/commit/d034256b3c18a900c96b058f6b65f4d775b1a767